code_samples/authorization/get_decision.mdx

```diff
@@ -123,6 +207,69 @@ public class GetDecisions {
 <TabItem value="js" label="Javascript">
 
 ```javascript
+import { PlatformClient } from '@opentdf/sdk/platform';
+import { AuthProviders } from '@opentdf/sdk';
+import { create } from '@bufbuild/protobuf';
+import { GetDecisionsRequestSchema, DecisionRequestSchema, Entity_CategorySchema } from '@opentdf/sdk/platform';
+
+async function main() {
+  const platformEndpoint = 'http://localhost:8080';
+  
+  // Assume you have an existing access token
+  const accessToken = 'your-access-token-here';
+
+  // Create auth provider with existing token
+  const authProvider = await AuthProviders.accessTokenAuthProvider({
+    accessToken: accessToken
+  });
+
+  // Create platform client
+  const platformClient = new PlatformClient({
+    platformUrl: platformEndpoint,
+    authProvider
+  });
+
+  // Get Decision using v1 API (bulk decisions)
+  const request = create(GetDecisionsRequestSchema, {
+    decisionRequests: [
+      create(DecisionRequestSchema, {
+        entityChains: [{
+          id: 'ec1',
+          entities: [{
+            id: 'entity-1',
+            entityType: {
+              case: 'clientId',
+              value: 'opentdf'
+            },
+            category: Entity_CategorySchema.SUBJECT
```

Comment:
I am not sure but it looks like this needs to be the `EntityCategory` enum and not `EntityCatergorySchema` enum

```suggestion
             category: Entity_Category.SUBJECT
```